### ISSUE TITLE
Weird layout of active class and inactive class on the student page

### ISSUE THREAD

**Issue by dailinshen** [dailinshen/coms4156_jumpstart#4]
It's really confusing to see a weird layout of the active class and inactive class on the student page. Right after a user logging in as a student, he/she shall see all the classes registered either with an open sign-in window or a close sign-in window. Before the student doing anything, all the classes on the page are styled correctly and neatly. Suppose there is a class for student to sign in, and suppose the student gives a valid secret code. Once he/she hits the submit button, the layout changes tremendously. Some classes with close sign-in window are removed to the bottom of the class he/she just signed in.

**Comment by dailinshen** [comment]
A big guess is that there is something wrong with the main_student.html page.

**Comment by dailinshen** [comment]
After I looked carefully into the page, I found there is a missing {% endif %}. Also, I moved the active == 0 up inside the first "for" loop statement. Because originally, the page was doing two times of for loop, which had only one if statement. I merged the two for loop together and the whole for loop is just being used to check the active status of each registered class. 

In another words, the issue is fixed.
